Yasgui/packages/yasr/src/ontotext-yasr.ts

```diff
@@ -0,0 +1,173 @@
+import Yasr, { Config } from "@triply/yasr";
+import { addClass, removeClass } from "@triply/yasgui-utils";
+
+export class OntotextYasr extends Yasr {
```

Comment:
I think we should use more generic names for our extensions here in the source code of the yasgui, e.g. `ExtendedYasr` or `EnhancedYasr` or whatever but witout Ontotex